hbase_sql/src/main/java/com/splicemachine/compactions/SpliceCompactionRequest.java

```diff
@@ -82,4 +102,26 @@ public void setRegion(HRegion region) {
         this.region = region;
     }
 
+    @Override
+    public void setOffPeak(boolean value) {
+        // We hijack setOffPeak because it is only called twice:
+        // 1. set to true in SpliceDefaultCompactionPolicy before compaction happens
+        // 2. set to false in HStore.finishCompactionRequest (hbase code)
+        // At those points, the value passed is irrelevant and is "only" used for logging
+        // purpose, so we can hijack it.
+        super.setOffPeak(value);
+        if (!value) {
+            afterExecute();
+        }
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
These two methods can be removed since they don't change the superclass implementation.

Reply:
Those are there as documentation to satisfy spotbugs.